src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -134,8 +138,8 @@ public PeriodicBackupRunner(DocumentDatabase database, ServerStore serverStore)
             bool skipErrorLog = false)
         {
             var now = SystemTime.UtcNow;
-            var lastFullBackup = backupStatus.LastFullBackupInternal ?? now;
-            var lastIncrementalBackup = backupStatus.LastIncrementalBackupInternal ?? backupStatus.LastFullBackupInternal ?? now;
+            var lastFullBackup = backupStatus.LastFullBackupInternal ?? _databaseWakeUpTime ?? now;
```

Comment:
Why do we need this?

Reply:
to prevent a case:
DB is idle and have timer to make a backup _for first time,_ 
In this case we was passing now as lastFullBackup => nextBackupOccurrence calculated as the next occurrence (according to cron expression).
For example, db is idle and has timer for _first backup_:
backup each 5 mins -> woke up on 00:05 -> lastfullbackup = now (00:05) -> nextBackupOccurrence(lastfullbackup) = 00:10